### ISSUE TITLE
Sidebar nav block config updates

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#1406]
## Update 8/5/2022
[Meeting notes](https://docs.google.com/document/d/1FnDTe3tBeK31wKyTdlujoel9d6X9XBT63s3vO_U0zEw/edit#heading=h.zhkufj9wt9vw)
Hide sidebar nav when off-canvas menu is present
- Use d-none classes in block configuration

## Motivation
I want the content of a page to be the first content displayed when viewing a page on mobile.

### Is your feature request related to a problem? Please describe.
Follow up ticket to https://github.com/az-digital/az_quickstart/issues/1202

## Proposed Resolution
- [x] Hide sidebar nav block on mobile
- [x] Remove block title's top margin
- [x] Move to second sidebar region

### Describe alternatives you've considered
- Keep location above content, collapse menu into "Section menu"
- Allow off canvas nav to expand beyond 2 levels, remove sidebar from main content area

**Comment by joshuasosa** [comment]
To clarify, will a solution _move_ the sidebar nav somewhere else while on mobile? Mobile users shouldn't have degraded access to resources.

**Comment by camikazegreen** [comment]
@joshuasosa Yes, the second and third (and more?) tiers of navigation will still be available. I missed the outcome of the last meeting, but I believe we are leaning towards enhancing the off canvas mobile nav to include deeper tiers of navigation, be more present and do a better job of highlighting where you are on the site when opened. 

More user testing is going to be done to make sure that this is the right approach, vs the other option we explored which was to collapse the sidebar navigation into an expandable menu that would still be at the top of the page.

**Comment by danahertzberg** [comment]
@joshuasosa Based on the outcome of [last week's workshop](https://docs.google.com/document/d/1hl1H6yEzjV-CpT1DNR7GImoVhVTi_oWgRdKkOXo6FeA/edit), we decided to put the sidebar nav in the off-canvas element rather than display it with body content. This ticket is one in a series of related tickets to achieve these goals. This will not be in a release without navigation appearing elsewhere.

**Comment by danahertzberg** [comment]
Description updated with new tasks

**Comment by akslay** [comment]
![Screen Shot 2022-08-10 at 3 11 41 PM](https://user-images.githubusercontent.com/52931751/184029412-4de49467-3b91-4cf1-ae07-e12145ad90b1.png)

Removing the top margin from the sidebar nav `h2` does have visual implications. Are we okay with this or not okay with this?

**Comment by danahertzberg** [comment]
@akslay I think this is okay to allow content editors to define their own spacing

**Comment by joshuasosa** [comment]
@camikazegreen @danahertzberg sounds good as long as off canvas nav supports unlimited levels!

@akslay I think it would be good to add a default margin top class to the block so it's not such a significant visual change. Even just mt-4 would be good.

**Comment by danahertzberg** [comment]
What if the block move to second sidebar and title removal by default did _not_ have a db update? And we encourage the changes via communications to existing site owners. But new sites would all have the new functionality?

**Comment by joshuasosa** [comment]
Yes! Keeping things as-is for existing sites and changing them for new sites sounds good to me!

**Comment by danahertzberg** [comment]
To reduce impact on existing sites and to marry nicely with the page title region changes, this ticket will only require removing the sidebar nav's block title top margin. By changing `mt-md-5` to `mt-0`.

**Comment by danahertzberg** [comment]
Will address as follow up to https://github.com/az-digital/az_quickstart/issues/1826

**Comment by joshuasosa** [comment]
> https://psychology.arizona.edu/apply-our-graduate-program

If you set the text on media to mb-5 and change the "Thank you" text to use an h4 class like the menu heading does, it looks aligned.
![image](https://user-images.githubusercontent.com/4400438/226784293-546632d4-bf54-46bf-883a-60534a78f011.png)

**Comment by trackleft** [comment]
Fixed in #4671
